about.html

```diff
@@ -9,7 +9,34 @@
src="assets/about/partners/VeCow.png" alt="VeCow" loading="lazy"></picture></div><div class="col-6 col-md-4 col-lg-3 logo-animate-container"><picture><source srcset="assets/about/partners/Yallvend@2x.webp" type="image/webp"><source srcset="assets/about/partners/Yallvend.png, assets/about/partners/Yallvend@2x.png 2x"><img class="logo-animate" src="assets/about/partners/Yallvend.png" alt="Yallvend" loading="lazy"></picture></div></div></div></section><section id="section-contact"><div class="container thin-container"><h2 class="section-title" data-i18n="about-contact">聯絡我們</h2><br><br><div class="form-wrapper row"><div class="col-12 col-lg-6"><h4 class="text-theme font-weight-bold" data-i18n="about-contact-message">現在，馬上就可以輕易地跟我們取得聯繫：</h4><br><form id="contact-form" action="https://formspree.io/myynnnqd" method="POST"><p style="margin-bottom: 0;" class="map-text" data-i18n="about-contact-name">姓名</p><input type="text" name="client_name" id="client_name" class="map-input-column" style="border: 0px;"><br><br><p style="margin-bottom: 0;" class="map-text" data-i18n="about-contact-email">電子信箱</p><input type="text" name="client_email" id="client_email" class="map-input-column" style="border: 0px;"><br><br><p style="margin-bottom: 0;" class="map-text" data-i18n="about-contact-content">內容</p><textarea name="client_content" id="client_content" rows="10" class="map-input-column"></textarea><br><br><input type="submit" value="Submit" class="map-submit-btn" data-i18n="[value]about-contact-send"></form></div><div class="col-12 col-lg-6"><br><br><br><iframe width="100%" height="300" frameborder="0" style="border:0" src="https://www.google.com/maps/embed/v1/place?key=AIzaSyCqolKTPGa4IxsURDmIJ6aL_7nOm2vYY_w&q=100台北市中正區市民大道三段2號10F" loading="lazy" allowfullscreen></iframe><br><br><div class="map-text row no-gutters align-items-center"><div class="col-auto"><picture><source srcset="assets/about/map-pin@2x.webp" type="image/webp"><source srcset="assets/about/map-pin.png, assets/about/map-pin@2x.png 2x"><img src="assets/about/map-pin.png" class="map-icon" alt="map-pin"></picture></div><div class="col"><!-- <p style="margin-bottom: 0;">FutureWard-Central 共同工作空間</p> --> <a class="text-theme" href="https://www.google.com.tw/maps/search/FutureWard-Central%20%E5%85%B1%E5%90%8C%E5%B7%A5%E4%BD%9C%E7%A9%BA%E9%96%93/" target="_blank" data-i18n="about-contact-address">台北市中正區市民大道三段 2 號 10 樓</a><p style="margin-bottom: 0;" data-i18n="about-contact-time">週一 ~ 週五10:00~17:00</p></div></div></div></div></div></section><footer><div class="container"><div class="row align-items-center"><div class="col-6 col-lg-3"><picture><source srcset="assets/common/logo-theme.webp" type="image/webp"><source srcset="assets/common/logo-theme.png, assets/common/logo-theme@2x.png 2x"><img src="assets/common/logo-theme.png" class="img-fluid" alt="logo"></picture></div><!-- <div class="col"></div> --><div class="col-auto ml-auto pr-5"><!-- <a class="footer-link" href="#" data-i18n="footer-recruit">徵才訊息</a>
```

Comment:
Why did you use Chinese identifier name?

Reply:
fixed